build.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash -e
 
-GECKO_SDK_VERSION="com.silabs.sdk.gecko_platform.v2.7.feature_root_2.7.7.202008240742-8"
-GECKO_SDK_URL="https://devtools.silabs.com/studio/v4/updates/binary/"
+GECKO_SDK_USER="SiliconLabs"
+GECKO_SDK_REPO="gecko_sdk"
+GECKO_SDK_SHA1="991121c706578c9a2135b6f75cc88856e8c64bdc" # Version 4.0
```

Comment:
Finally, they moved to Git(Hub) (agan)!

Would it be possible to rename this to `GECKO_SDK_VERSION`. That would fit better when referencing a tag or branch name as well.

Reply:
I picked the SHA1 intentionally, because Tags and Branches may change the commit they are pointing to. Furthermore, the Github download links look a little bit different for tags and branches.

But if you still want it to point to branches, I'll try to come up with a proper solution ;-)

Reply:
I think it's fine for now then. I could only imagine that, if they use proper versioning some time, we could stick to the 4.x branch or so.

Reply:
The idea was to have a build script that always creates the same `dist` folder no matter when executed. (Of course, things may go wrong and objcopy could create different outputs in the future ... but at least I tried :D)